### PR TITLE
Create International PL landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Creates a new International Professional Learning page at https://code.org/professional-learning/international.
- Works with LTR and RTL languages
- Fully responsive

**Jira ticket:** [ACQ-1409](https://codedotorg.atlassian.net/browse/ACQ-1409)

----

localhost code org_3000_professional-learning_international

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -1618,6 +1618,43 @@
   yw_page_top_heading: "Young Women in Computer Science"
   yw_page_top_desc: "Become a CS Ambassador and be a part of closing the gender gap in computer science."
 
+  intl_pl_page:
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:1621]: @mgc1194 using the new namespace pattern here and loooove it! So much easier to implement 🙌

File: pegasus/sites.v3/code.org/public/professional-learning/international.haml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: International Professional Resources
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/professional-learning/international.haml]: does the title need to be translated?

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/professional-learning/international.haml]: Ideally yes, but now that you mention it I don't think I've seen any titles translated on Pegasus pages 🤔 Do you know if this is possible?

Comment (bethanyaconnor) [reply pegasus/sites.v3/code.org/public/professional-learning/international.haml]: I guess not :/ that's a bit of a bummer

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/professional-learning/international.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/professional-learning/international.haml]: I did some more digging and found out how to do it! 🎉 ([here's an example](https://github.com/code-dot-org/code-dot-org/blob/e7c0b75a16c73ba132d3b616913abcd894401aee/pegasus/sites.v3/code.org/public/index.haml#L13-L14) on the homepage) 

I'll tag you on the commit shortly.

File: pegasus/sites.v3/code.org/public/professional-learning/international.haml [hunk 3]
```diff
@@ -1,9 +1,11 @@
 ---
-title: International Professional Resources
 theme: responsive_full_width
 set_dir: true
 ---
 
+:ruby
+  @header["title"] = hoc_s("intl_pl_page.page_title")
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/professional-learning/international.haml:7]: @bethanyaconnor can you take a look at this?

Comment (bethanyaconnor) [reply pegasus/sites.v3/code.org/public/professional-learning/international.haml:7]: Woo! Probably not needed on every page but this is great on pages that are aimed at international users!

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/professional-learning/international.haml (hunk 3 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/professional-learning/international.haml:7]: Agreed! 🙌

**Pull request merged**